config/db.config.js

```diff
@@ -0,0 +1,47 @@
+export const HOST = 'localhost';
+export const USER = 'postgres';
+export const PASSWORD = 'Barbaris179$';
+export const DB = 'postgres';
+export const dialect = 'postgres';
+export const pool = {
```

Comment:
and it you will have time investigate the "repository" pattern for managing layered architecture (something, you started to implement)